pages/partials/head.ejs

```diff
@@ -3,6 +3,9 @@
 <meta name="viewport" content="width=device-width, initial-scale=1">
 <title>PrairieLearn</title>
 <link href="/node_modules/bootstrap/dist/css/bootstrap.min.css" rel="stylesheet" />
+<% if(locals.darkmode == 1){%>
+<link href="/node_modules/bootstrap/dist/css/bootstrap1.min.css" rel="stylesheet" />
+<% }%>
```

Comment:
I'm worried about including random bootstrap themes. These have a bad history of becoming unsupported. Is there any more standard way to get darkmode with bootstrap?